File: tests/better_volume_test.py
import numpy as np
from taflow import BetterVolume
def test_better_volume_lifecycle():
    s=BetterVolume(np.array([],float),np.array([],float),np.array([],float),np.array([],float));s.extend([2],[1],[1.5],[10]);assert s.value is not None;s.reset();assert len(s)==0

File: crates/taflow-core/src/indicators/better_volume_test.rs
use super::better_volume::BetterVolume;
#[test]
fn lifecycle() {
    let mut s = BetterVolume::new(2).unwrap();
    assert!(s.append(2.0, 1.0, 1.5, 10.0).is_none());
    assert!(s.append(3.0, 1.0, 2.0, 20.0).is_some());
    s.reset();
    assert!(s.value().is_none());
}
